prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -587,7 +587,7 @@ ssize_t efa_rdm_ep_post_handshake(struct efa_rdm_ep *ep, struct efa_rdm_peer *pe
 	fi_addr_t addr;
 	ssize_t ret;
 
-	addr = peer->efa_fiaddr;
+	addr = peer->conn->fi_addr;
```

Comment:
Can you clarify the behavior of `conn->fi_addr` after you split the AV into explicit and implicit? AFAICT, when you post handshake, the peer can be only in implicit av at that time when it was unknown. The `conn->fi_addr` is then the index in implicit av only? Will it be updated to the explicit AV index after a future fi_av_insert happens? Or there will be separate field for the explicit av index in the peer? I lost my collection for the design details somehow.